### ISSUE TITLE
Visualize the results without fitting the model

### ISSUE THREAD

**Issue by bhishanpdl** [DistrictDataLabs/yellowbrick#979]
Let's say I have to visualize a confusion matrix.

I can use yellowbrick and use the LogisticRegression and visualize like this:
https://www.scikit-yb.org/en/latest/api/classifier/confusion_matrix.html
```
from sklearn.datasets import load_iris
from sklearn.model_selection import train_test_split as tts
from sklearn.linear_model import LogisticRegression
from yellowbrick.classifier import ConfusionMatrix

iris = load_iris()
X = iris.data
y = iris.target
classes = iris.target_names

X_train, X_test, y_train, y_test = tts(X, y, test_size=0.2)

model = LogisticRegression(multi_class="auto", solver="liblinear")

iris_cm = ConfusionMatrix(
    model, classes=classes,
    label_encoder={0: 'setosa', 1: 'versicolor', 2: 'virginica'}
)

iris_cm.fit(X_train, y_train)
iris_cm.score(X_test, y_test)

iris_cm.show()
```

But, most of the times I use scikit-learn and I already have confusion matrix:
For example:
```
cm = np.array([[56750,   114],
              [   95,     3]])
```

Can we now simply use this result in YELLOWBRICK, give label names visualize it?

**Comment by bbengfort** [comment]
Hi, @bhishanpdl if you update to the latest version of Yellowbrick, v1.0.1 -- we've recently added the ability for the visualizers to detect if the model is fitted or not, this will ensure that your model is not refit if you've already fitted it elsewhere and will deterministically draw the CM, even if you already have it in another data structure. Perhaps this would be the simplest thing to do:

```
viz = ConfusionMatrix(model, is_fitted=True)
viz.fit(X_train, y_train)
viz.score(X_test, y_test)
viz.show()
```

We don't have any support or functions for drawing a confusion matrix directly without the model (though if you'd like to create a PR to add one, we'd be happy to take a look at it!) However, currently you can hack Yellowbrick as follows:

```
iris_cm = ConfusionMatrix(
    model, classes=classes,
    label_encoder={0: 'setosa', 1: 'versicolor', 2: 'virginica'}
)

iris_cm. confusion_matrix_ = cm
iris_cm.draw()
iris_cm.show()
```

See [confusion_matrix.py#L227](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/yellowbrick/classifier/confusion_matrix.py#L227) for more details on what is drawn.

Of course, we don't guarantee this will work forever, but it might be a simple way to get your confusion matrices drawn right now. 

Thanks for using Yellowbrick!
